content/v2.0/cloud/get-started.md

```diff
@@ -128,8 +128,6 @@ The primary differences between InfluxDB OSS 2.0 and InfluxDB Cloud 2.0 are:
 - Retrieving data from a file based CSV source using the `file` parameter of the
   [`csv.from()`](/v2.0/reference/flux/functions/csv/from) function is not supported;
   however you can use raw CSV data with the `csv` parameter.
-- Multi-organization accounts and multi-user organizations are currently not
-  available in {{< cloud-name >}}.
```

Comment:
Where is this repeated?

Reply:
Line 127: `- {{< cloud-name "short" >}} instances are currently limited to a single organization with a single user.`

Reply:
Nice catch, @pierwill! 
Maybe we could swap "instances" with "accounts"...and/or make singular and positive? An InfluxDB Cloud account currently supports a single....